app/src/org/commcare/tasks/ProcessAndSendTask.java

```diff
@@ -466,6 +467,18 @@ protected int getSuccessfulSends() {
         return successes;
     }
 
+    protected String getLabelForFormsSent() {

```

Comment:
Ah I see, you were just pulling out the code for doing this that was previously in `FormAndDataSyncer`. So in that original implementation, the mistake there was actually that there is no reason to pass a numeric parameter in the case that you are using `sync.success.sent.singular` key. So you should just remove the parameter in that case, and change that key back to what it was before